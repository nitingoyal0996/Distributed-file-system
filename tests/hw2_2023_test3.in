create f1
create f2
create f3
create f4
ls
rm f1
ls
exit
