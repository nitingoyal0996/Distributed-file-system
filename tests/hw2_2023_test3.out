#### File system information:
Number of blocks          : 256
Block size (Bytes)        : 128
Number of inodes          : 16
inode size (Bytes)        : 16
inodes per block          : 8
Free bitmap offset        : 2
Free bitmap size (blocks) : 2
Inode table offset        : 4
Inode table size (blocks) : 2
Max blocks per file       : 2
Data blocks offset        : 6
Data block size (blocks)  : 250
Raw block layer layout: (B: boot, S: superblock, F: free bitmap, I: inode, D: data
0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345
BSFFIIDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[cwd=0]%
[5]:./
[1]:f1
[1]:f2
[1]:f3
[1]:f4
[cwd=0]%
[cwd=0]%
[4]:./
[1]:f2
[1]:f3
[1]:f4
[cwd=0]%
